projects/pdf-player-wc/src/app/app.module.ts

```diff
@@ -21,7 +21,7 @@ import { SunbirdPdfPlayerComponent } from '../../../sunbird-pdf-player/src/lib/s
     SunbirdPlayerSdkModule
   ],
   providers: [],
-  entryComponents: [SunbirdPdfPlayerComponent]
+  // entryComponents: [SunbirdPdfPlayerComponent]
```

Comment:
Why this is commented

Reply:
entryComponents is longer required with the Ivy compiler and runtime.